### PR TITLE
NIAD-1188: Readme file for VnP testing

### PR DESCRIPTION
**Pull request by bartek-sarul**
## Description

Please explain the changes you have made here.

## Jira Ticket

Please add a link to the related Jira ticket (if any).

## Checklist

These are items (excluding GitHub Checks) which should be confirmed before a branch is ready to merge.

- [ ] Acceptance Criteria met
- [ ] Commit messages are meaningful
- [ ] Manually tested
- [ ] Self-reviewed your code
- [ ] Code reviewed from two other developers
- [ ] If your pull request depends on any other, please link them in the description
- [ ] main branch is passing/green on CI
- [ ] Add/update any relevant documentation

### CODE REVIEW

File: mesh/mesh.sh [hunk 1]
```diff
@@ -40,12 +40,17 @@ inbox() {
...
 send() {
   local body
+  local count
   body="$1"
-  curl "${CURL_FLAGS}" -X POST "https://${HOST}/messageexchange/${MAILBOX_ID}/outbox" \
-    --cert "${OPENTEST_ENDPOINT_CERT}" --key "${OPENTEST_ENDPOINT_PRIVATE_KEY}" -H "Authorization: ${TOKEN}" -d "${body}" \
-    -H "Mex-From: ${MAILBOX_ID}" -H "Mex-To: ${TO_MAILBOX}" -H "Mex-WorkflowID: ${WORKFLOW_ID}" \
-    -H 'Content-Type:application/octet-stream' -H 'Mex-MessageType: DATA'  -H 'Mex-FileName: test-filename.txt' -H 'Mex-Version: 1.0' \
-    -H 'Mex-ClientVersion: 1.0' -H 'Mex-JavaVersion: 1.7.0_60' -H 'Mex-OSArchitecture: Windows 7' -H 'Mex-OSName: x86' -H 'Mex-OSVersion: 6.1'
+  count="$2"
```
Comment (andrewwa-kainos) [inline mesh/mesh.sh:45]: can we have `count` be optional, defaulting to 1?

File: vnp/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Volumetric and Performance testing
+
+There are 2 ways how to put load on the Lab Results adaptor in order to measure it's performance.
```
Comment (andrewwa-kainos) [inline vnp/README.md:3]:
```suggestion
There are 2 ways to put load on the Lab Results adaptor in order to measure its performance.
```

File: vnp/README.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+## Sending EDIFACT to MESH
+
+Use the MESH Client script to send the same file multiple times to a given MESH mailbox. Details about the script can be found [here](../mesh/README.md).
+
+Example:
+
+    ./mesh.sh send gp_mailbox "@../src/intTest/resources/edifact/pathology.edifact.dat" 3 
+
+will send 3 times the content of a file located at `../src/intTest/resources/edifact/pathology.edifact.dat` to `gp_mailbox` MESH mailbox.
```
Comment (andrewwa-kainos) [inline vnp/README.md:19]:
```suggestion
will send the content of a file located at `../src/intTest/resources/edifact/pathology.edifact.dat` to `gp_mailbox` MESH mailbox 3 times.
```

File: vnp/README.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+## Sending EDIFACT to `Inbound MESH Queue`
+
+Use the ["A" application](https://github.com/fmtn/a) which is an AMQP client allowing you to send messages directly to `Inbound MESH Queue`. Java 11 required.
+
+Example:
+
+    java -jar a-1.5.0-jar-with-dependencies.jar -A -b "amqps://admin:pwd@my-mq.com:5671" -p "@../src/intTest/resources/edifact/pathology.edifact.dat" -c 5 lab_results_mesh_inbound
+
+will send 5 times the content of a file located at `../src/intTest/resources/edifact/pathology.edifact.dat` to `lab_results_mesh_inbound` queue on `my-mq.com` broker listening on port `5671` connecting using secured AMQP protocol (amqp<span style="color:yellow">**s**</span>), user name `admin` and password `pwd`
```
Comment (andrewwa-kainos) [inline vnp/README.md:29]:
```suggestion
will send the content of a file located at `../src/intTest/resources/edifact/pathology.edifact.dat` 5 times to the `lab_results_mesh_inbound` queue on `my-mq.com` broker listening on port `5671` connecting using secured AMQP protocol (amqp<span style="color:yellow">**s**</span>), user name `admin` and password `pwd`
```

**Review by andrewwa-kainos — APPROVED** [review]

**Review by saileshp-kainos — APPROVED** [review]

**Review by antonlebedjko — APPROVED** [review]

**Pull request merged**